### PR TITLE
Creating a common script for cluster density and node density* workloads

### PR DESCRIPTION
**Pull request by shahsahil264**
@paigerube14

### CODE REVIEW

**Comment by shahsahil264** [comment]
@krishvoor Sorry, the last PR had conflicts and got closed by mistake

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

File: ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-commands.sh
Comment (afcollins) [inline ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-commands.sh:1]: I think it makes sense to use an agnostic step name. I think it is confusing if the step called "cluster-density-v2" also runs node-density tests.

**Pull request closed without merging**